src/components/QuranReader/TranslationView/TranslationViewCell.module.scss

```diff
@@ -1,7 +1,26 @@
 @use "src/styles/breakpoints";
 
 .highlightedContainer {
-  background: var(--color-background-alternative-faded);
+  --color-gradient-transition: 196, 196, 196;
+
+  [data-theme="dark"] & {
+    --color-gradient-background-dark: 150, 125, 225;
+    background: radial-gradient(ellipse farthest-corner at -360% -360%, rgba(var(--color-gradient-background-dark), 0.25) 50%, rgba(var(--color-gradient-transition), 0) 90%);
+  }
+
+  [data-theme="light"] & {
+    --color-gradient-background-light: 60, 120, 180;
+    background: radial-gradient(ellipse farthest-corner at -360% -360%, rgba(var(--color-gradient-background-light), 0.45) 50%, rgba(var(--color-gradient-transition), 0) 90%);
+  }
+
+  background-color: var(--color-background-alternative-faded);
+  margin: 0 calc(var(--spacing-medium) * -1);
+  border-radius: var(--spacing-xsmall);
```

Comment:
We have pre-set tokens for `border-radius`
<img width="459" alt="Screen Shot 2021-12-20 at 18 51 27" src="https://user-images.githubusercontent.com/15169499/146763124-622f1e3a-019c-4d24-a47a-ff5279bff1e7.png">